docs/development/release-process.rst

```diff
@@ -64,26 +64,24 @@ Release Process
 Creating a new release
 ----------------------
 
-#. On the current pip ``master`` branch, generate a new ``AUTHORS.txt`` by
-   running ``invoke generate.authors`` and commit the results.
-#. On the current pip ``master`` branch, make a new commit which bumps the
-   version in ``pip/__init__.py`` to the release version and adjust the
-   ``CHANGES.txt`` file to reflect the current date. The release version should
-   follow a YY.N scheme, where YY is the two digit year, and N is the Nth
-   release within that year.
-#. On the current pip ``master`` branch, generate a new ``NEWS.rst`` by running
-   ``invoke generate.news`` and commit the results.
-#. Create a signed tag of the ``master`` branch of the form ``YY.N`` using the
-   command ``git tag -s YY.N``.
-#. Checkout the tag using ``git checkout YY.N`` and create the distribution
-   files using ``python setup.py sdist bdist_wheel``.
+#. Checkout the current pip ``master`` branch.
+#. Generate a new ``AUTHORS.txt`` (``invoke generate.authors``) and commit the
+   results.
+#. Bump the version in ``pip/__init__.py`` to the release version and commit
+   the results. Usually this involves dropping just the ``.devN`` suffix on the
+   version.
+#. Generate a new ``NEWS.rst`` (``invoke generate.news``) and commit the
+   results.
+#. Create a signed tag at the current commit, of the form ``YY.N``
```

Comment:
We don't use signed tags any more, because I don't have signing ability on Windows, and we don't have a project signing key anyway.

Reply:
I'd signed the tag. (I also sign every commit).

I don't mind if we remove this signing requirement as well.